openerp/addons/base/res/res_company.py

```diff
@@ -229,13 +229,14 @@ def _company_default_get(self, cr, uid, object=False, field=False, context=None)
         if not context:
             context = {}
         proxy = self.pool.get('multi_company.default')
+        user = self.pool.get('res.users').browse(cr, SUPERUSER_ID, uid, context=context)
```

Comment:
Yes, it will work as it's not private in the sense of Java for example, but there is still an "internal use" concept associated with it in Python. It's just that we don't seem to care about it very much in odoo code... ;-)